tools/gulp/tasks/lint.ts

```diff
@@ -19,7 +21,13 @@ const materialOutPath = join(buildConfig.outputDir, 'packages', 'material');
 /** Path to the output of the CDK package. */
 const cdkOutPath = join(buildConfig.outputDir, 'packages', 'cdk');
 
-task('lint', ['tslint', 'stylelint', 'madge']);
+/** Path for the Github owners file. */
+const ownersFilePath = '.github/CODEOWNERS';
+
+/** Paths that should be ignored when checking for owners coverage. */
+const ignoreOwners = ['/node_modules/**', '/dist/**', '/.git/**', '/.idea/**'];
```

Comment:
This doesn't really cover all the paths (e.g. there's `/.vscode` which should also be excluded). Consider populating the list from the `.gitignore` instead?